components/rmrk/Profile/ProfileDetail.vue

```diff
@@ -313,22 +313,7 @@ export default class Profile extends mixins(PrefixMixin) {
     this.checkId()
 
     try {
-      this.$apollo.addSmartQuery('collections', {
-        query: collectionList,
-        client: this.urlPrefix,
-        manual: true,
-        // update: ({ nFTEntities }) => nFTEntities.nodes,
-        loadingKey: 'isLoading',
-        result: this.handleCollectionResult,
-        variables: () => {
-          return {
-            account: this.id,
-            first: this.first,
-            offset: this.collectionOffset,
```

Comment:
I remove `first` for two reasons：
1. It limited that just the first 20 collections can be shown, which is an incorrect logic.
2. This collection page has not yet introduced pagination. I think the count of collections is so little that we can introduce pagination in the future.